tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/internal/types/Tensors.java

```diff
@@ -0,0 +1,158 @@
+package org.tensorflow.internal.types;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.StringJoiner;
+import org.tensorflow.Tensor;
+import org.tensorflow.ndarray.NdArray;
+import org.tensorflow.ndarray.Shape;
+
+/**
+ * Tensor helper methods.
+ */
+public final class Tensors {
+
+  /**
+   * Prevent construction.
+   */
+  private Tensors() {
+  }
+
+  /**
+   * Equivalent to {@link #toString(Tensor, Integer) toString(tensor, null)}.
+   *
+   * @param tensor a tensor
+   * @return the String representation of the tensor
+   */
+  public static String toString(Tensor tensor) {
+    return toString(tensor, null);
+  }
+
+  /**
+   * @param tensor   a tensor
```

Comment:
Can you add some description?  I.e. how will maxWidth change the output?  I assume it controls the use of the `..`.s?

Reply:
I've added a description. Please review.